README.md

```diff
@@ -24,6 +24,18 @@ custom:
     distributionFolder: client/dist # (Optional) The location of your website. This defaults to client/dist
     indexDocument: index.html # (Optional) The name of your index document inside your distributionFolder. Defaults to index.html
     errorDocument: error.html # (Optional) The name of your error document inside your distributionFolder. Defaults to error.html
+    objectMetadata: # (Optional) S3 object metadata (HTTP headers) for hosted files
+      ALL_OBJECTS: # (Optional) List of headers to set for all files in bucket
+        - name: Content-Language # HTTP header name
```

Comment:
Too ambiguous IMO. Can we use `headerName`?

Reply:
Works for me!